src/test/java/io/jenkins/plugins/uleska/toolkitscanner/UleskaToolkitScannerTest.java

```diff
@@ -0,0 +1,158 @@
+package io.jenkins.plugins.uleska.toolkitscanner;
+
+import hudson.model.TaskListener;
+import io.jenkins.plugins.uleska.scan.ScanApi;
+import io.jenkins.plugins.uleska.scan.ScanException;
+import io.jenkins.plugins.uleska.toolkit.Toolkit;
+import io.jenkins.plugins.uleska.toolkit.ToolkitLocator;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Spy;
+
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
```

Comment:
No tests use assertNotNull, unnecessary import

Reply:
Removed this.